code/src/index.css

```diff
@@ -5,9 +5,103 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  display: flex;
+  justify-content: center;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+  text-align: center;
+  margin: 5%;
+}
+
+.form {
+  display: flex;
+  flex-wrap: wrap;
+  min-width: 320px;
+  max-width: 400px;
+  padding: 4% 5%;
+  margin: 5%;
+  background-color: #d3d3d363;
+  border: 1px solid;
+  box-shadow: 8px 8px;
+}
+
+.post {
+  min-width: 320px;
+  max-width: 400px;
+  padding: 5%;
+  margin: 5%;
+  border: 1px solid;
+  box-shadow: 8px 8px;
+}
+
+h3 {
+  margin: 0;
+}
+
+.date {
+  margin: 0;
+}
+
+.like-container {
+  display: flex;
+}
+
+.post-details {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #b0b0b0;
+  margin-top: 5%;
+}
+
+.like-button {
```

Comment:
Just wanted to add a suggestion. You could change the cursor to pointer instead like this:
`cursor: pointer;`
Or add a hover effect to the button so that you can see that the button is clickable. You could also add it to the .submit-button :)